specification/cluster/put_component_template/ClusterPutComponentTemplateRequest.ts

```diff
@@ -67,7 +66,11 @@ export interface Request extends RequestBase {
      * @server_default 30s */
     master_timeout?: Duration
 
-    cause?: string
```

Comment:
Instead of commenting, let's remove it entirely. Git can be used to check history, if needed.